src/components/application_manager/include/application_manager/app_launch/app_launch_data_json.h

```diff
@@ -132,12 +134,12 @@ class AppLaunchDataJson : public AppLaunchDataImpl {
   /**
    * @return pointer to AppLaunch data block in Json file
    */
-  Json::Value& GetSavedApplicationDataList() const;
+  Json::Value& GetSavedApplicationDataList(Json::Value& dictionary) const;
 
   /**
    * @return pointer to AppLaunch records block in Json file
    */
-  Json::Value& GetApplicationData() const;
+  Json::Value& GetApplicationData(Json::Value& dictionary) const;
```

Comment:
@LitvinenkoIra Is it a part of public interface?

Reply:
@AByzhynar no, this is a private method